services/datamanager/builtin/builtin.go

```diff
@@ -225,6 +229,10 @@ type resourceMethodMetadata struct {
 	MethodMetadata data.MethodMetadata
 }
 
+func (r resourceMethodMetadata) String() string {
+	return fmt.Sprintf("Resource Name: %s, Method Params: %s, Method Metadata: [%v]", r.ResourceName, r.MethodParams, r.MethodMetadata)
```

Comment:
Nice use of using the underlying String() method on method metadata! Since the way that we're packaging it up is specific to our implementation and the user doesn't need to know about it, how would you feel about having this adjust to have the output in your PR description be:

```
capture frequency for [Api: rdk:component:camera, Resource Name: fake_camera, Method Name: ReadImage,  Method Params: map[mime_type:image/jpeg]] is set to 1.00Hz and will sync
```

This goes from highest-level to lowest-level scope of what we're capturing, and rounds the float.

Reply:
Sounds good! Should be done :)